src/components/patterns/modals/modal.tsx

```diff
@@ -48,7 +48,7 @@ const Modal = ({children, isOpen, onClose, labelledBy}: ModalProps) => {
               <button
                 type="button"
                 onClick={onClose}
-                className={"absolute right-50 top-50 text-white flex"}
+                className={"absolute right-50 top-50 text-white flex hocus:underline"}
```

Comment:
I assume this is a tailwind modification for hover and focus?

Reply:
Yes it is.